lib/driver-helper.js

```diff
@@ -528,3 +528,23 @@ export async function dismissAlertIfPresent( driver ) {
 		return false;
 	}
 }
+
+export async function clear_cache( driver ) {
+	let url = await driver.getCurrentUrl();
+	if ( url.startsWith( 'data:' ) === false && url !== 'about:blank' ) {
```

Comment:
When I printed the url here, most of the time the url wouldn't be either `data:` nor `about:blank`. Does that indicate some unexpected behavior?

Reply:
Not necessarily, for example in the `dismissAllAlerts` function we use that page 
https://github.com/Automattic/wp-e2e-tests/blob/3654d16829e8fff9483756d14430cc69c5a9fd42/lib/driver-manager.js#L317